src/main/java/org/ow2/proactive/catalog/rest/controller/CatalogObjectController.java

```diff
@@ -242,7 +240,7 @@ public CatalogObjectMetadataList create(
             } catch (IOException ioe) {
                 throw new RuntimeException(ioe);
             }
-            return ResponseEntity.status(status).build();
+            return new ResponseEntity<>(status);
```

Comment:
I prefer to use the builder .... ResponseEntity.status(status).build();

Reply:
ResponseEntity.status(status).build(); return a Response<Void> which doesn't compile due to the returne Response<List<CatalogObjectMetadata>>